dockers/docker-sonic-bmp/docker_init.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+
+# Generate supervisord config file
+mkdir -p /etc/supervisor/conf.d/
+
+udp_server_ip=$(ip -j -4 addr list lo scope host | jq -r -M '.[0].addr_info[0].local')
+hostname=$(hostname)
+# Generate the following files from templates:
+# port-to-alias name map
+sonic-cfggen -d -t /usr/share/sonic/templates/rsyslog.conf.j2 \
+    -a "{\"udp_server_ip\": \"$udp_server_ip\", \"hostname\": \"$hostname\"}" \
+    > /etc/rsyslog.conf
+
+# Make the script that waits for all interfaces to come up executable
+chmod +x /usr/bin/start.sh
```

Comment:
We could ensure `+x` during build time. Do we really need it in runtime?

Reply:
fixed